packages/kbn-openapi-generator/README.md

```diff
@@ -0,0 +1,201 @@
+# OpenAPI Code Generator for Kibana
+
+This code generator could be used to generate runtime types, documentation, server stub implementations, clients, and much more given OpenAPI specification.
+
```

Comment:
Thank you for including the detailed description here in the README as well! 

WRT wider adoption and use, should we surface as a documentation set on docs.elastic.dev, or have an entry in the [Kibana Developer Guide](https://docs.elastic.dev/kibana-dev-docs/getting-started/welcome)?

Reply:
Yes, that's a good idea 👍 I've also been thinking about making a broader announcement, like sending an email about the codegen features to attract more early adopters. But starting with documentation is definitely a solid first step.